Wander/app/src/main/java/com/teamwan/wander/FeedbackFragmentChart.java

```diff
@@ -69,11 +79,19 @@ private void initialiseFragment(){
         if (titleString.length()>20) {
             title.setTextSize(26);
         }
-        if (chartData instanceof LineData) {
-            chart.setData((LineData) chartData);
-            chart.invalidate();
+        if (chartTypeLine) {
```

Comment:
I don't know if I follow, but if you mean `if (titleString.length()>20)` and `if (chartTypeLine)` I disagree. The titles might be changed or a chart could be added so which titles are long might change.